sdk-libs/macros/docs/CLAUDE.md

```
@@ -52,21 +52,21 @@ Documentation for the rentfree macro system in `light-sdk-macros`. These macros
-        - LightCompressible       <- traits/light_compressible.md (combines all)
+    +-- Uses trait derives (account/):
+        - HasCompressionInfo      <- account/has_compression_info.md
+        - CompressAs              <- account/compress_as.md
+        - Compressible            <- account/compressible.md
+        - CompressiblePack        <- account/compressible_pack.md
+        - LightCompressible       <- account/light_compressible.md (combines all)
 ```
 
 ## Related Source Code
 
 ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)